src/Controls/src/Core/VisualElement/VisualElement.cs

```diff
@@ -1269,9 +1272,14 @@ protected override void OnChildRemoved(Element child, int oldLogicalIndex)
 		{
 			base.OnChildRemoved(child, oldLogicalIndex);
 
-			if (child is View view)
+			if (child is VisualElement visualElement)
 			{
-				view.ComputedConstraint = LayoutConstraint.None;
+				visualElement.MeasureInvalidated -= OnChildMeasureInvalidated;
```

Comment:
I think the problem is when a VisualElement goes away it doesn't unsubscribe like it does here, `visualElement.MeasureInvalidated` making the page leak